app/utils/pool-utils.ts

```diff
@@ -59,7 +59,7 @@ export class PoolUtils {
     }
 
     public static isOfferWindows(offer: string) {
-        return /^.*Windows.*$/.test(offer);
```

Comment:
to fix your problem do it the way it was but with the i for case insensitive

/windows/i.test(offer)

also mark the function to return a boolean explicitly